src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/PersonTest.kt

```diff
@@ -8,46 +8,23 @@ import uk.gov.justice.digital.hmpps.personrecord.client.model.court.commonplatfo
 import uk.gov.justice.digital.hmpps.personrecord.client.model.court.commonplatform.DefendantAlias
 import uk.gov.justice.digital.hmpps.personrecord.client.model.court.commonplatform.PersonDefendant
 import uk.gov.justice.digital.hmpps.personrecord.client.model.court.commonplatform.PersonDetails
-import uk.gov.justice.digital.hmpps.personrecord.client.model.court.event.LibraHearingEvent
-import uk.gov.justice.digital.hmpps.personrecord.client.model.court.libra.Name
 import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.PNCIdentifier
 import uk.gov.justice.digital.hmpps.personrecord.model.person.Person
 import uk.gov.justice.digital.hmpps.personrecord.model.types.ContactType.HOME
 import uk.gov.justice.digital.hmpps.personrecord.model.types.ContactType.MOBILE
-import uk.gov.justice.digital.hmpps.personrecord.test.randomFirstName
-import uk.gov.justice.digital.hmpps.personrecord.test.randomLastName
+import uk.gov.justice.digital.hmpps.personrecord.test.randomName
 import uk.gov.justice.digital.hmpps.personrecord.test.randomPnc
 import java.time.LocalDate
 
 internal class PersonTest {
 
-  @Test
-  fun `should map libra hearing to person`() {
-    val dateOfBirth = LocalDate.now()
-    val inputPncId = randomPnc()
```

Comment:
PNC was the only one of these fields not being checked in [LibraCourtEventListenerIntTest.kt](https://github.com/ministryofjustice/hmpps-person-record/pull/481/files#diff-cf0f8b04383625b7d74ade36be628822baecac9dc6759609b31acfcd1d2b8dcc)